  Comments: 
    + currently enable automatic libtoolize and remove acinclude.m4
      since looks like a copy of an old libtool.m4

--- fribidi-0.10.4/Makefile.old	2005-03-19 16:28:14.000000000 +0100
+++ fribidi-0.10.4/Makefile	2005-03-19 16:28:47.000000000 +0100
@@ -17,11 +17,12 @@
 build: build-stamp
 build-stamp: unpack-stamp
 	install -d build/fribidi
-#	cd $(TARDIR) && aclocal
-#	cd $(TARDIR) && autoheader
-#	cd $(TARDIR) && libtoolize -c --force
-#	cd $(TARDIR) && automake -i --foreign
-#	cd $(TARDIR) && autoconf
+	rm $(TARDIR)/acinclude.m4
+	cd $(TARDIR) && aclocal
+	cd $(TARDIR) && autoheader
+	cd $(TARDIR) && libtoolize -c --force
+	cd $(TARDIR) && automake -i --foreign
+	cd $(TARDIR) && autoconf
 
 	cd build/fribidi && ../../$(TARDIR)/configure  --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info
 	cd build/fribidi && make $(MAKE_DEFINES)
